src/components/FlowRunCreateForm.vue

```diff
@@ -18,33 +15,23 @@
         </p-text-input>
       </p-label>
 
-      <p-label label="Message (Optional)">
-        <p-textarea v-model="stateMessage" placeholder="Created from the Prefect UI" />
-      </p-label>
-
-      <p-label label="Tags (Optional)">
-        <p-tags-input v-model="tags" :options="deploymentTags" />
-      </p-label>
-
-      <p-label v-if="workPoolName && !workPool?.isPushPool" :label="workQueueComboboxLabel">
-        <WorkPoolQueueCombobox v-model:selected="workQueueName" :work-pool-name="workPoolName" />
-      </p-label>
-
-      <div class="flow-run-create-form__retries">
-        <p-label label="Retries (optional)">
-          <p-number-input v-model="retries" :min="0" />
-        </p-label>
+      <template v-if="hasParameters">
+        <p-divider />
 
-        <p-label label="Retry Delay (optional)">
-          <p-number-input v-model="retryDelay" :min="0" append="Seconds" />
-        </p-label>
-      </div>
+        <p-content>
+          <h3>
+            {{ localization.info.parameters }}
+          </h3>
 
-      <p-divider v-if="deployment.parameters" />
+          <SchemaInput v-model="parameters" v-model:input-type="parametersInputType" :schema="deployment.parameterOpenApiSchema">
+            <template #button-group>
+              <p-button-group v-model="parametersInputType" :options="parametersInputTypeOptions" small />
+            </template>
+          </SchemaInput>
+        </p-content>
+      </template>
 
-      <h3 class="flow-run-create-form__section-header">
-        Start
-      </h3>
+      <h3>Start</h3>
```

Comment:
I feel like this doesn't need to be an `h3` anymore now that there isn't a general heading. Should we just wrap the `p-button-group` in a `p-label`? That would be more inline with the rest of the form

Reply:
Yea let me give that a shot

Reply:
So that makes the text here smaller, and @billpalombi had a request that it be sized the same as the `Parameters` heading (which is how I chose h3); open to achieving that however you think is best

Reply:
ah okay no that's fine leave it as is